test.sh

```diff
@@ -94,3 +94,6 @@ if [[ $OS != "fedora" ]]; then $RUN $PIP install -U setuptools; fi
 
 # Run tests
 $RUN pytest -vv tests --cov atomic_reactor "$@"
+
+echo "To run tests again:"
+echo "$RUN pytest -vv tests --cov atomic_reactor "
```

Comment:
Seems unrelated to the pulp_publish changes.

Reply:
It is unrelated... just a convenience for the future